graphics/src/MeshManager.cc

```diff
@@ -822,6 +822,85 @@ void MeshManager::CreateCamera(const std::string &_name, float _scale)
   mesh->RecalculateNormals();
 }
 
+void MeshManager::CreateEllipsoid(const std::string &_name,
+                                  const ignition::math::Vector3d &_radii,
+                                  const unsigned int _rings,
+                                  const unsigned int _segments)
+{
+  if (this->HasMesh(_name))
+  {
+    return;
+  }
+
+  Mesh *mesh = new Mesh();
+  mesh->SetName(_name);
+  this->dataPtr->meshes.insert(std::make_pair(_name, mesh));
+
+  ignition::math::Vector3d vert;
+  SubMesh subMesh;
+
+  const double umin = -M_PI / 2.0;
```

Comment:
Windows is dumb about the M_PI constant.

An easy way to get it (without changing defines and causing issues):

`const double pi = std::acos(-1);`

Reply:
Looking at the rest of the methods `IGN_PI` is the solution.